motoman_mh5_support/launch/robot_interface_streaming_mh5.launch

```diff
@@ -13,12 +13,10 @@
 	
   <!-- controller: Controller name (fs100 or dx100) -->
   <arg name="controller"/>
-  <arg name="version0"/>
 
 	<rosparam command="load" file="$(find motoman_mh5_support)/config/joint_names_mh5.yaml" />
 
 	<include file="$(find motoman_driver)/launch/robot_interface_streaming_$(arg controller).launch">
 		<arg name="robot_ip"   value="$(arg robot_ip)" />
-        <arg name="version0"   value="$(arg version0)" />
 	</include>
 </launch>
```

Comment:
It started failing afterward, but since the roslaunch tests were failing on Travis (due to catkin tools bug and the issue above), I didn't notice.  This is why I want to pull this in before other merges.